wormchain/x/wormhole/keeper/instantiate_allowlist.go

```diff
@@ -0,0 +1,22 @@
+package keeper
```

Comment:
This new data stored should be added to wormhole module's GenesisState type. It should then be populated in genesis.go's InitGenesis() and ExportGenesis() functions.